### ISSUE TITLE
Line Breaks Before End Notes

### ISSUE THREAD

**Issue by dylan-k** [WaltersArtMuseum/journal-thewalters-org#28]
On a page like https://journal.thewalters.org/?p=3170&preview=true often a linebreak will occur before the end note, creating an orphan. If possible, prevent linebreak before end note.

`<sup><a href="[#edn-12](https://journal.thewalters.org/?p=3170&preview=true#edn-12)">[12]</a></sup`

**Comment by dylan-k** [comment]
The css for this is probable `white-space: nowrap;` but the html for end notes may need a class added, so that the css can be applied only to end note links, not other kinds of links.

**Comment by rbowler-wam** [comment]
@dylan-k How do you want me to text this?

**Comment by dylan-k** [comment]
This is on hold, so that it can be built using the new development server, after #24
